backend/src/configs/hands.clj

```diff
@@ -1,8 +1,7 @@
-(ns configs.hand)
+(ns configs.hands)
 
-(defn ini-hand
-  "Generates de initial hand (can't be empty)"
-  []
+(def default-hand
```

Comment:
Why change from `defn` to `def`?

Reply:
It wasn't a function.
Since I had to change them all anyway, I thought I could change that too.